modules/appeals_api/app/workers/appeals_api/weekly_error_report.rb

```diff
@@ -4,13 +4,14 @@
 
 module AppealsApi
   class WeeklyErrorReport
+    include ReportRecipientsReader
     include Sidekiq::Worker
     # Only retry for ~48 hours since the job is run weekly
     sidekiq_options retry: 16, unique_for: 48.hours
 
     def perform(to: Time.zone.now, from: 1.week.ago.beginning_of_day)
-      recipients = Settings.modules_appeals_api.reports.weekly_error.recipients
-      if enabled?
+      recipients = load_recipients(:error_report_weekly)
+      if enabled? && !recipients.empty?
```

Comment:
Actually I think it'd read better to be `recipients.present?` instead of `!recipients.empty?`. Kinda surprised Rubocop didn't pick that one up, actually.